packages/experimental/src/global/js/package-settings.cmn.js

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright IBM Corp. 2020, 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+const defaults = {
+  prefix: 'exp',
+  // by default only released components are set to true
+  component: {
+    AboutModal: false,
+    ActionBar: false,
+    APIKeyDownlaoder: false,
+    APIKeyModal: false,
+    BreadcrumbWithOverflow: false,
+    Card: false,
+    ContextHeader: false,
+    EmptyState: false,
+    ExampleComponent: false,
+    ExportModal: false,
+    ImportModal: false,
+    ModifiedTabs: false,
+    Notifications: false,
+    PageActionItem: false,
+    PageHeader: false,
+    RemoveDeleteModal: false,
+    SidePanel: false,
+    TagSet: false,
+    Tearsheet: false,
+    TearsheetNarrow: false,
+    TearsheetShell: false,
+    WebTerminal: false,
+  },
+
+  // feature level flags
+  feature: {
+    'a-new-feature': false,
+  },
+};
+
+const setAllOn = (flags, enabled) => {
```

Comment:
```suggestion
const setAll = (flags, enabled) => {
```
According to the enabled arg, this is either setting all on or setting all off.